src/VisualStudio/IntegrationTest/TestUtilities/Common/Comparison.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.VisualStudio.IntegrationTest.Utilities.Common
     internal static class Comparison
     {
         public static bool AreStringValuesEqual(string str1, string str2)
-            => string.IsNullOrEmpty(str1) == string.IsNullOrEmpty(str2)
+            => string.IsNullOrEmpty(str1) && string.IsNullOrEmpty(str2)
             || str1 == str2;
```

Comment:
Seems like this expression is easier to figure out:

```
(str1 ?? "") == (str2 ?? "")
```

Reply:
sure, I can change that on the other PR.